src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/BedSearchServiceTest.kt

```diff
@@ -533,7 +534,14 @@ class BedSearchServiceTest {
             name = "Room Characteristic Name",
           ),
         ),
-        overlaps = mutableListOf(),
+        overlaps = mutableListOf(
```

Comment:
I suspect this test change should live in the same commit that changed the behaviour, it's hard to see at a commit level how this fits together. I wonder if this could also now be squashed into https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/908/commits/0fd2cf49ca919b109d7156204ba5bd4f1a6cc783#diff-a54610f6ab7620b26539197f1ff5789d74e9966cb68c712ba3c193a3b0e9cffbR536 ?

Reply:
I've done some significant squashing and re-ordering to attempt to reconcile this issue too.